### DISCUSSION TITLE
[Regression]request.form.get("form_data") is not populating with url params in jinja template functions after upgrading tag 1.2.0

### DISCUSSION THREAD

**Discussion by saurabnigam** [apache/superset#18316] [Q&A / Help]
In tag 1.0.0  , functions defined via jinja templating received url_params in request.form.get("form_data")
After upgrading to tag 1.2.0 request.form.get("form_data") is returning empty array. What changed? Is something broken or this parameters passed somewhere else now

**Comment by junlincc** [comment]
@saurabnigam apologize for the regression, unfortunately we are not actively supporting jinja template anymore
@amitmiran137 @villebro if we have touched this area and caused regression from 1.0-1.2, please push a fix? 🙏
